feat: Qdrant integration (#2422)
DataDog/integrations-extras: qdrant/README.md

# Qdrant

<img width="1139" alt="Qdrant Datadog Dashboard" src="https://raw.githubusercontent.com/DataDog/integrations-extras/master/qdrant/images/qdrant_dashboard.png">

## Overview

> [Qdrant][1] is a high-performance vector search engine and vector database. Use the Datadog integration to get metrics in real-time to monitor your Qdrant deployment.

Author: [`1d86a8c` (#2422)](https://github.com/DataDog/integrations-extras/pull/2422/commits/1d86a8c704ef0c6346fa261a8f81328c80b7121f)

For a full list of supported metrics, see the **Metrics** section below.

## Setup

The Qdrant check is not included in the [Datadog Agent][2] package, so you need to install it.

### Installation

For Agent v7.21+ / v6.21+, follow the instructions below to install the Qdrant check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.

1. Run the following command to install the Agent integration:

   ```shell

>    datadog-agent integration install -t qdrant==<INTEGRATION_VERSION>

Reviewer: This version should be included in the command

   ```

2. Configure your integration similar to core [integrations][4].

### Configuration

1. Edit the `qdrant.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][7] to start collecting your Qdrant [metrics](#metrics).

2. [Restart the Agent][9]

## Validation

Run the [Agent's status subcommand][10] and look for `qdrant` under the Checks section.

## Data Collected

### Metrics

See [metadata.csv][11] for a list of metrics provided by this check.

### Events

The Qdrant check does not include any events.

### Service Checks

See [service_checks.json][13] for a list of service checks provided by this integration.

## Troubleshooting

> Need help? Contact [Datadog support][12].

Reviewer: This should be the qdrant support team

# Qdrant

## Overview

> [Qdrant][1] is a high-performance vector search engine and vector database. Use the Datadog integration to get metrics in real-time to monitor your Qdrant deployment.
> 
> - You can monitor the performance of REST and gRPC interfaces with metrics such as total requests, total failures, time to serve etc to identify potential bottlenecks and mitigate them.
> - You can find information about the readiness of the cluster, and deployment(total peers, pending operations etc) to gain insights into your Qdrant deployment.

Reviewer: ```suggestion
[Qdrant][1] is a high-performance vector search engine and vector database. With this integration, you can get real-time metrics to monitor your Qdrant deployment within Datadog including: 

- The performance of REST and gRPC interfaces with metrics such as total requests, total failures, and time to serve to identify potential bottlenecks and mitigate them.
- Information about the readiness of the cluster, and deployment (total peers, pending operations, etc.) to gain insights into your Qdrant deployment.
```

> 

Author: Do you mean the mentioned features? Yes. These metrics are available.
